modules/grafana/scripts/local_dashboard_directory_prompt.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+#
+# Prompts user for the relative path to dashboards directory.
+# To avoid this prompt, set LOCAL_DASHBOARD_DIRECTORY when you run make.
+read -erp "Enter relative path to dashboards directory: " dir
```

Comment:
Bash has a selector prompt built-in, if you want to do it that way:
```sh
select dir in **/dashboards/; do
    echo -n "$dir"
done
```

Reply:
Thanks, this worked well. Added this with an option to enter manually in the new revision.